lib/response.js

```diff
@@ -40,6 +40,14 @@ Response.prototype.redirect = function (u) {
 	// In a timeout because page.js did it, but I believe it is better
 	// to let the current route handler finish its stuff before starting
 	// another round of routing, otherwise we could get some weird behavior.
+
+	// Discard the status code argument if it exists
+	if (arguments.length === 2) {
```

Comment:
Would rather not use arguments for this.  Just define both arguments and type check:

```javascript
Response.prototype.redirect = function (status, url) {
  if (typeof status === 'string') {
    url = status;
    status = 302;
  }
  // ...
};
```